examples/nomad-consul-ami/setup_ubuntu16.sh

```diff
@@ -7,13 +7,7 @@ SCRIPT=`basename "$0"`
 #echo "[INFO] [${SCRIPT}] Setup git"
 #sudo apt install -y git
 
-# Using Docker CE directly provided by Docker
-echo "[INFO] [${SCRIPT}] Setup docker"
-cd /tmp/
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
-sudo apt-get update
-apt-cache policy docker-ce
+# Desired Nomad drivers - Docker and Java
+sudo apt install docker.io -y
```

Comment:
`apt` often has very out of date versions of Docker. The original option is what the official Docker documentation recommends for Ubuntu, so we should stick with it.